### PR TITLE
Fix typos around permissions

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: JitsiMediaDevices.js [hunk 1]
```diff
@@ -156,7 +156,19 @@ class JitsiMediaDevices {
                 }
 
                 Promise.all(promises).then(
-                    r => resolve(r.every(Boolean)),
+                    r => {
+                        const grantedPermissions = r.every(permissionStatus => {
```
Comment (webintoit) [inline JitsiMediaDevices.js]: `isPermissionsGranted`

File: JitsiMediaDevices.js [hunk 2]
```diff
@@ -156,7 +156,19 @@ class JitsiMediaDevices {
...
+                            // The status attribute is deprecated, and state
+                            // should be used instead, but check both for now
+                            // for backwards compatibility.
+                            const grantStatus = permissionStatus.state
+                                || permissionStatus.status;
+
+                            return grantStatus === 'granted';
```
Comment (webintoit) [inline JitsiMediaDevices.js]: `granted` Should be moved to upper level constant

File: JitsiMediaDevices.js [hunk 3]
```diff
@@ -156,7 +156,19 @@ class JitsiMediaDevices {
                 }
 
                 Promise.all(promises).then(
-                    r => resolve(r.every(Boolean)),
+                    r => {
```
Comment (webintoit) [inline JitsiMediaDevices.js]: what is `r` ?  is `response ` better here ?

Comment (saghul) [reply JitsiMediaDevices.js]: response shortened as r is a generally used convention

**Pull request merged**